### PR TITLE
Add ariaLabel to See more link

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

File: src/ViewModel/SeeMoreLink.php [hunk 1]
```diff
@@ -15,11 +15,12 @@ final class SeeMoreLink implements ViewModel
 
     protected $url;
 
-    public function __construct(Link $link, bool $isInline = false)
+    public function __construct(Link $link, bool $isInline = false, string $ariaLabel = null)
```
Comment (nlisgo) [inline src/ViewModel/SeeMoreLink.php:18]: I think we should move the ariaLabel to the second parameter.

But also, seeing it like this does make me wonder whether we should incorporate ariaLabel directly into the link which could mean that we gain much wider support!

File: tests/src/ViewModel/SeeMoreLinkTest.php [hunk 2]
```diff
@@ -15,21 +15,25 @@ public function it_has_data()
...
         $this->assertSame($data['isInline'], $link['isInline'], 'The isInline property should be true');
         $this->assertSame($data, array_merge($link->toArray(), ['isInline' => true]));
     }
 
     public function viewModelProvider() : array
     {
         return [
-            [new SeeMoreLink(new Link('something', 'http://google.com'))],
+            'minimal' => [new SeeMoreLink(new Link('something'))],
+            'link only' => [new SeeMoreLink(new Link('something', 'http://google.com'))],
```
Comment (nlisgo) [inline tests/src/ViewModel/SeeMoreLinkTest.php:35]: Have modified minimal as url is optional and introduced a link only

**Review by nlisgo — APPROVED** [review]
LGTM. Just modified one test. See recent commits.

File: tests/src/ViewModel/SeeMoreLinkTest.php (hunk 2 above)
Comment (viktoria-mahmud) [reply tests/src/ViewModel/SeeMoreLinkTest.php]: The 'url' is required for SeeMoreLink component.

File: src/ViewModel/SeeMoreLink.php [hunk 3]
```diff
@@ -12,13 +12,14 @@ final class SeeMoreLink implements ViewModel
...
 
     protected $name;
-
     protected $url;
+    protected $ariaLabel;
 
     public function __construct(Link $link, bool $isInline = false)
     {
         $this->name = $link['name'];
         $this->url = $link['url'];
```
Comment (nlisgo) [inline src/ViewModel/SeeMoreLink.php:21]: If link has url as optional but SeeMoreLink requires it then we should trigger an error in this function if `$link['url']` is not set.

Comment (nlisgo) [reply src/ViewModel/SeeMoreLink.php:21]: We could pair on this on Monday. Or, if we want to get this in today. We could revert my changes and then create a ticket to address this as a followup.

Comment (viktoria-mahmud) [reply src/ViewModel/SeeMoreLink.php:21]: I created a test to check the required URL. Can you check it when you have time?

File: src/ViewModel/SeeMoreLink.php [hunk 4]
```diff
@@ -12,13 +12,14 @@ final class SeeMoreLink implements ViewModel
...
+        $this->ariaLabel = $link['ariaLabel'];
```
Comment (smoqadam) [inline src/ViewModel/SeeMoreLink.php:22]: For `ariaLabel` we also need to check if it's set before assign it to `$this->ariaLabel`

**Review by smoqadam — APPROVED** [review]

**Pull request merged**